music.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :publish_date
+
+  def initialize(on_spotify, publish_date)
+    super(publish_date)
```

Comment:
The first parameter on parent initilizer is: "id"

### def initialize(id, publish_date, archived: false)

You may need to send a - nil - value to ensure

### @id = id || Random.rand(1..1000) 

work properly as

### super(nil, publish_date)